modules/swagger-codegen/src/main/resources/csharp/Project.mustache

```diff
@@ -88,7 +88,10 @@
     {{/netStandard}}
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="**\*.cs"/>
+    <Compile Include="Api\**\*.cs"/>
```

Comment:
@voydz @wing328 I verified the change breaks solutions in Rider and Visual Studio. I opened #5605 to resolve the issue, but I don't have Xamarin installed and no space on my drive to install it. Could one of you verify the change works as expected in Xamarin.

Screenshots below to clarify the issue, where the solution doesn't display namespaces specified with apiPackage or modelPackage.

Visual Studio solution on master:
<img width="282" alt="windows_10" src="https://cloud.githubusercontent.com/assets/109659/25903555/c1133270-356a-11e7-9771-f0f5f1fd53bb.png">

Rider solution on master:
<img width="328" alt="rider_1_0_eap" src="https://cloud.githubusercontent.com/assets/109659/25903611/f100919e-356a-11e7-92fa-cbcde9d16735.png">